packages/genesys-spark-components/src/genesys-spark-utils/get-closest-element.ts

```diff
@@ -0,0 +1,23 @@
+//https://inindca.atlassian.net/browse/COMUI-2673
+export function closestPassShadow(
```

Comment:
Could you explain a bit more about the idea behind this function name? I get what it does but I'm just wondering if there's a different way to describe it.

Reply:
I updated the function name to match the name of the file, `getClosestElement`. I do have a follow up ticket, COMUI-2673 to centralize this function and remove duplication